pkg/sysprom/initialize.go

```diff
@@ -146,36 +147,34 @@ func SetupSystemPrometheus() error {
 	return nil
 }
 
-// waitForResouces waits for resources to get into running state
-func waitForResources(w *InitConfig) error {
-	var testResouces = []Resource{
-		{name: "kube-state-metrics", namespace: "pf9-monitoring", kind: "deployment"},
-		{name: "catalog-operator", namespace: "pf9-olm", kind: "deployment"},
-		{name: "olm-operator", namespace: "pf9-olm", kind: "deployment"},
-		{name: "packageserver", namespace: "pf9-olm", kind: "deployment"},
-		{name: "prometheus-operator", namespace: "pf9-operators", kind: "deployment"},
-		{name: "node-exporter", namespace: "pf9-monitoring", kind: "daemonset"},
+// waitForCRD waits for CRD's to be created
+func waitForCRD(w *InitConfig) error {
+	var crds = []string{
+		"prometheuses.monitoring.coreos.com",
+		"prometheusrules.monitoring.coreos.com",
+		"servicemonitors.monitoring.coreos.com",
+		"alertmanagers.monitoring.coreos.com",
 	}
 
 	// Set timeout value for wait
 	timeout := time.After(10 * time.Minute)
```

Comment:
please move these timeouts to command line args/ENV variable

Reply:
Made requested changes